src/mantarray_desktop_app/process_monitor.py

```diff
@@ -70,11 +70,13 @@ def __init__(
         ],  # pylint: disable=unsubscriptable-object
         the_lock: threading.Lock,
         boot_up_after_processes_start: bool = False,
+        skip_load_firmware_file: bool = False,
```

Comment:
I forget where I read this, but somewhere I think I remember seeing that ideally kwarg bools should be true if it's not too much trouble. that generally leads to shorter variable names: `load_firmware_file=True`

Reply:
I may be misremembering but I thought we specifically wanted them to be False and that's why `boot_up_after_processes_start` in the line above is False. The cmd line arg for skipping boot up stores true when passed but we specifically flip that value on line 232 in `main.py` due to it defaulting to False here, even though the default behavior of the app is to automatically boot up after the processes start

Reply:
it's a somewhat fuzzy guideline.  `boot-Up_after_processes_start` is definitely clearer and more concise than `skip_boot_up_after_processes_start`.  And I agree that the command line arg should be called `skip` since we need the default behavior to be "do run it".  

I was just specifically talking about this `skip_load_firmware_file` vs `load_firmware_file` in this method here.  But it's not a big deal.  Just something to consider